ai-ml/jupyterhub/variables.tf

```diff
@@ -31,23 +31,51 @@ variable "secondary_cidr_blocks" {
   type        = list(string)
 }
 
-#  Domain name is public so make sure you use a unique while deploying
+
+# NOTE: You need to use private domain or public domain name with ACM certificate
+# This website doc will show you how to create free public domain name with ACM certificate for testing purpose only
+# Example of public domain name(<subdomain-name>.<domain-name>.com): eks.jupyter-doeks.dynamic-dns.com
+variable "jupyter_hub_auth_mechanism" {
+  type        = string
+  description = "Allowed values: cognito, dummy"
+  default     = "cognito"
+}
+
+variable "jupyter_notebook_support" {
+  type        = string
+  description = "Allowed values: cpu, gpu"
+  default     = "cpu"
+}
+
+#  Domain name is public so make sure you use a unique while deploying, Only needed if auth mechanism is set to cognito
 variable "cognito_custom_domain" {
   description = "Cognito domain prefix for Hosted UI authentication endpoints"
   type        = string
-  default     = "eks-jupy"
```

Comment:
Since you are changing the default value, Please check if this name is used in the docs.

Reply:
This is not used anywhere in the docs